.github/actions/generate-coverage/tests/test_scripts.py

```
@@ -0,0 +1,116 @@
+from __future__ import annotations
+
+import subprocess
+from pathlib import Path
+
+from shellstub import StubManager
+
+
+def run_script(
+    script: Path, env: dict[str, str], *args: str
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)